app/components/LoginContainer.tsx

```diff
@@ -15,14 +17,31 @@ export default function LoginContainer({
     <Container
       fluid
       style={{ height: "100vh", overflow: "hidden" }}
-      className="bg-dark"
+      className=""
     >
       <div className="d-flex justify-content-center align-content-center h-100">
         <div className="align-self-center p-5 bg-white rounded text-center">
```

Comment:
Not every day you see the web-dev challenge of "center this div horizontally and vertically" done in practice. ✨ very nice!